domain/domain.go

```diff
@@ -356,12 +358,15 @@ type etcdBackend interface {
 }
 
 // NewDomain creates a new domain. Should not create multiple domains for the same store.
-func NewDomain(store kv.Storage, lease time.Duration) (d *Domain, err error) {
+func NewDomain(store kv.Storage, lease time.Duration, factory pools.Factory) (d *Domain, err error) {
+	minCapacity := 1               // minCapacity for the sysSessionPool size
+	maxCapacity := 200             // maxCapacity for the sysSessionPool size
```

Comment:
Should we enlarge maxCapacity to 500 or 1000? 
https://github.com/pingcap/tidb/blob/master/server/server.go#L126